mlrun/runtimes/function.py

```diff
@@ -1626,3 +1635,25 @@ def extend_handler(base_handler):
         return "", extend_handler(handler)
 
     return split_handler[0], extend_handler(split_handler[1])
+
+
+def _resolve_nuclio_runtime_python_image(
+    mlrun_version: str = None, python_version: str = None
```

Comment:
```suggestion
    mlrun_client_version: str = None, python_version: str = None
```
to distinguish between mlrun api which always is >= 1.3.0 and mlrun clients which may be older